OlatunjiYso/WEconnect
# Bug #158394821 fix reviews bugs

#### What does this PR do?
- This PR fix the bug that does not make the changes made to reviews reflect immediately

#### Description of Task to be completed?
- When a review is deleted, the effect is seen immediately
- when a review is updated, it comes live
- when a review is added, the effect is seen immediately

#### How should this be manually tested?
- clone the repo
- navigate to localhost:3000
- add reviews, modify them, deleted them and observe the immediate response.

#### What are the relevant pivotal tracker stories?
[Pivotal Tracker #158394821](https://www.pivotaltracker.com/n/projects/2153356)
